eng/Microsoft.Extensions.targets

```diff
@@ -25,7 +26,12 @@
   <ItemGroup>
     <PackageReference
         Update="Microsoft.Maui.Graphics"
-        Version="$(_MicrosoftMauiGraphics)" />
+        Version="$(_MicrosoftMauiGraphics)" 
+    />
+    <PackageReference
```

Comment:
Shouldn't this just be on WinUI target? should this have a Condition on the ItemGroup?

Reply:
I think it's fine since it's just an `Update`